test/infrastructure/docker/examples/clusterclass-quickstart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: "clusterclass-quickstart"
+  namespace: default
+  labels:
+    cni: kindnet
+spec:
+  clusterNetwork:
```

Comment:
If we want to have a minimal example we can drop the entire clusterNetwork

Reply:
Oh I wasn't aware of that. I thought the config is optional :). Just resolve :)

Reply:
Although it might be a good idea to use the same kubectl apply calico here as in the other quickstart. Would also simplify the quickstart by not requiring CRS. But your decision.

Reply:
To use calico we'll need to get the kubeconfig with the associated sed commands etc. for MacOS. I think I'll leave it as is for now, but it might be a good idea to revisit once we have a cleaner networking story for quick start. There might be some way to configure kindnet that I'm not aware of.

Reply:
Ah good point, didn't think about the kubeconfig issue.

I'm not sure if the networking story will change as it's not really a problem of the quickstart but of how CAPI works overall (if I understood the reference right).

Then let's just leave it as is. I think eventually at some point we can extend our current quickstart guide to cover both cases (with and without ClusterClass) but this will take some time.